pkg/launch/launchable.go

```diff
@@ -17,9 +17,13 @@ type LaunchableID string
 
 func (l LaunchableID) String() string { return string(l) }
 
+type LaunchableVersionID string
```

Comment:
Makes sense to me. But why is `LaunchableVersion` its own struct with an `ID` field? I have the same criticism of it: calling it a "version identifier" is strange. I really just want to call it the "version". But that's not relevant to this PR, which is fine to ship.